client/src/components/Toasts/Toasts.scss

```
@@ -0,0 +1,157 @@
+.toasts {
+  position: fixed;
+  top: 60px;
+  right: 20px;
+  width: 250px;
+  z-index: 9999;
+
+  &__notice-wrap {
+    position: absolute;
+  }
+}
+
+
+@keyframes toastFadeIn {
+  0% { opacity: 0; }
+  100% { opacity: 1; }
+}
+
+@keyframes toastFadeOut {
+  0% {
+    opacity: 1;
+    max-height: 250px;
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  50% {
+    opacity: 0;
+    max-height: 250px;
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  100% {
+    opacity: 0;
+    max-height: 0;
+    margin-bottom: 0;
+  }
+}
+
+
+.toast {
+  margin: 0 0 map_get($spacers, sm) 0;
+  border-radius: 3px;
+  font-family: inherit;
+  font-size: inherit;
+  word-wrap: break-word;
+  max-height: 250px;
+  border: 0;
+  border-left: 3px solid;
```

Will need to be 4px if the border-radius is changing to 4px.

```suggestion
  border-left: 4px solid;
```

---

I'll update the designs as this isn't the case there and works different in practice.